README.md

```diff
@@ -11,6 +11,9 @@ The following compression codings are supported:
 
   - deflate
   - gzip
+  - br (brotli)
+
+**Note** Brotli provides better and faster compression then gzip or deflate, but is supported only since Node.js versions v11.7.0 and v10.16.0.
```

Comment:
Everything I am reading about br says that compression speeds are significantly slower than gzip. Can you link a benchmark (ideally using Node.js itself, since that is what we are using) that shows that br is faster than gzip and deflate? I'm only asked because I just know from history a claim like this is the readme is going to attract attention and so we could either have the benchmarks to show it or just not even make such a statement in the readme of this module.

Reply:
I’ll try to add a link tomorrow or the day after, as we are moving to a different apartment tomorrow.
Anyway there are lots of resources, and I did a benchmark myself on friday morning using node.js. The results were outstanding for text and binary alike. Compressed size was 10% than the gzip (comparing level 4 against gzip’s best!), and speeds were between 14 to 27 times faster, depending on compression levels.

Actually Zstd (facebook) is even faster (especially in decompression) and have even better compression ratios, but it has no up to date native implementation for node, and of course no built in support.
The benchmarks in node with the existing zstd modules basically suck (speed-wise).

Some links:

https://cran.r-project.org/web/packages/brotli/vignettes/brotli-2015-09-22.pdf

http://www.instantshift.com/2018/03/02/gzip-vs-brotli-compression/

https://expeditedsecurity.com/blog/nginx-brotli/

https://medium.com/oyotech/how-brotli-compression-gave-us-37-latency-improvement-14d41e50fee4

Reply:
I've updated the docs.